src/cljs/nnts2/note/events.cljs

```diff
@@ -2,19 +2,29 @@
   (:require [re-frame.core :as re-frame]
             [nnts2.note.api :as api]))
 
+
 (enable-console-print!)
 
+
 (re-frame/reg-event-fx
  :note-submit
  (fn [_ event]
-   (api/create-note (rest event))))
+   {:http-xhrio (api/create-note-req-map (first (rest event)))}))
```

Comment:
Putting a name on something makes it much easier to understand rather than referring to it by position. Destructuring the arguments as `[_ [_ note-data]]` lets us do away with `first` and `rest`.